deploy/apache/sites-available/privacyidea.conf

```diff
@@ -11,6 +11,14 @@
 		AllowOverride None
 	</Directory>
 
+        # Yubico servers use /wsapi/2.0/verify as the path in the
+        # validation URL. Some tools (e.g. Kolab 2fa) let the 
+        # user/admin change the api host, but not the rest of
+        # the URL. Let's redirect the api URL to privacyideas
+        # /ttype/yubikey.
```

Comment:
Cornelius Kölbel <notifications@github.com> writes:

>> @@ -11,6 +11,14 @@
>>  		AllowOverride None
>>  	</Directory>
>>  
>> +        # Yubico servers use /wsapi/2.0/verify as the path in the
>> +        # validation URL. Some tools (e.g. Kolab 2fa) let the 
>> +        # user/admin change the api host, but not the rest of
>> +        # the URL. Let's redirect the api URL to privacyideas
>> +        # /ttype/yubikey.
>
> Please put this in comments.

Ok.

I don't use nginx, but we might want to provide the same hint
there. I'll look at the nginx documentation for hints.

There's also something in nginx which is missing in the apache config.
nginx redirects all http-requests to https. Do we want that for apache
as well?

Jochen

-- 
The only problem with troubleshooting is that the trouble shoots back.